.chloggen/elasticsearch-profiles-downsampling.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add stratified downsampling to the profiles support in the elasticsearch exporter
```

Comment:
If this gets merged before the next release, I don't think this needs a changelog entry, as it goes within the entry to support profiles export.

Reply:
Reverted the commit. I added it to satisfy the CI, because I can't set labels.